core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala

```diff
@@ -128,8 +128,6 @@ class HadoopRDD[K, V](
 
   protected val jobConfCacheKey: String = "rdd_%d_job_conf".format(id)
 
-  protected val inputFormatCacheKey: String = "rdd_%d_input_format".format(id)
```

Comment:
This is a part of developer API. I don't think we can just remove this out.

Reply:
Ok, Let's reserve it.